src/lib.rs

```diff
@@ -64,12 +64,18 @@ impl<'a> From<Model<'a>> for Session {
 //     Init
 // ------ ------
 
-fn init(url: Url, orders: &mut impl Orders<Msg<'static>, GMsg>) -> Init<Model<'static>> {
-    orders.send_msg(Msg::RouteChanged(url.try_into().ok()));
-    Init::new_with_url_handling(
-        Model::Redirect(Session::new(storage::load_viewer())),
-        UrlHandling::None,
-    )
+fn before_mount(_url: Url) -> BeforeMount {
+    BeforeMount::new()
+        .mount_point("app")
+        .mount_type(MountType::Takeover)
+}
```

Comment:
I have added loading... text to index.html which will be overwritten by the app.  

The good: users on slow connection will see the loader instead of a blank screen
The bad: is the app loads quickly enough, the screen will flash (happens on localhost)

I don't have a strong opinion here. I can remove this if that is better.

Reply:
Ok, don't remove the loading text. 
But delete `.mount_point("app")` pls. 

> The bad: is the app loads quickly enough, the screen will flash (happens on localhost)

We can add fade-in animation to make it less aggressive, but we don't need to make it more complex, your choice.

Reply:
ive left it as is for now